hr_employee_document/security/security.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).-->
+<odoo>
+    <record id="access_hr_employee_system_user_override" model="ir.model.access">
+        <field name="name">hr.employee system user override</field>
+        <field name="model_id" ref="hr.model_hr_employee" />
+        <field name="group_id" ref="base.group_user" />
+        <field name="perm_read" eval="1" />
```

Comment:
How do model access rule prevail when they have the same group? Could `access_hr_employee_system_user` interfere with these one?

Reply:
With this change it works as we wish by applying the rules correctly as the tests show. It is true that it is something "strange" and I don't know if it can have side effects. What do you think @pedrobaeza ?